### PR TITLE
No saved events list screen: scrollable

### PR DESCRIPTION
**Pull request by zoeabryant**
## Pre-flight check-list

Helps with https://github.com/redbadger/pride-london-app/issues/217 and https://github.com/redbadger/pride-london-app/issues/212

Before raising a pull request

* [ ] Documentation
* [x] Unit tests

## Pre-merge check-list

* [ ] Link to Trello ticket/GitHub issue (If applicable) https://github.com/redbadger/pride-london-app/issues/217 and https://github.com/redbadger/pride-london-app/issues/212
* [ ] Any risky areas identified
* [ ] Test plan provided
* [ ] Tester approved

## Tester check-list

* [ ] Tested on multiple devices / OS versions (Test on the physical device(s) you have access to, otherwise use BrowserStack):

  * [x] Galaxy S8 (Android 7.0/8.0)
  * [ ] Galaxy S7 (Android 6.0)
  * [ ] iPhone X (iOS 11.x)
  * [ ] iPhone 8 (11.x)
  * [ ] iPhone 7 (iOS 10.x)

  Optional:

  * [ ] Google Pixel 2
  * [ ] Galaxy S8 Plus
  * [ ] Galaxy S7 Edge
  * [ ] Galaxy S6
  * [x] iPhone 7+
  * [x] iPhone 6

Accessibility Testing (If applicable)

* [ ] Text-to-Voice (Android: Voice Assistant / iOS: Speak Selection)
* [x] Large Text (=<200%)
* [x] Landscape Screen orientation

Weak connection testing (If applicable)

* [ ] Airplane mode
* [ ] 2G/3G Network Settings

### CODE REVIEW

File: src/screens/SavedEventListScreen/NoSavedEvents.js [hunk 1]
```diff
@@ -52,6 +54,9 @@ const styles = StyleSheet.create({
     flex: 3,
     marginHorizontal: 12
   },
+  pad: {
+    marginBottom: 24 // padding when you scroll to bottom
```
Comment (frigus02) [inline src/screens/SavedEventListScreen/NoSavedEvents.js]: Looks like in other views we generally use 12px for the bottom padding.

See:
* https://github.com/redbadger/pride-london-app/blob/5ee29f70bb4169cbab17c593cdf471c107b6a1cd/src/screens/EventDetailsScreen/component.js#L191
* https://github.com/redbadger/pride-london-app/blob/3b20821e87dbd13b1cf2be73bc9e3b4ec5387b14/src/screens/SponsorScreen/component.js#L155
* https://github.com/redbadger/pride-london-app/blob/3b20821e87dbd13b1cf2be73bc9e3b4ec5387b14/src/screens/FilterModal/component.js#L117

File: src/screens/SavedEventListScreen/NoSavedEvents.js [hunk 2]
```diff
@@ -52,6 +54,9 @@ const styles = StyleSheet.create({
     flex: 3,
     marginHorizontal: 12
```
Comment (frigus02) [inline src/screens/SavedEventListScreen/NoSavedEvents.js]: Instead of a fixed `marginHorizontal: 12` we should use the `ContentPadding` component here.

**Comment by declanslevin** [comment]
Looks good!

File: src/screens/SavedEventListScreen/NoSavedEvents.js (hunk 1 above)
Comment (zoeabryant) [reply src/screens/SavedEventListScreen/NoSavedEvents.js]: it just looked a little small to me, but fair enough! Will change

**Pull request merged**